### PR TITLE
Add stan4j

### PR DESCRIPTION
**Pull request by IRus**
http://stan4j.com/

### CODE REVIEW

**Comment by akullpp** [comment]
I didn't find a lot about the popularity of this library on the net. Also it is closed source, albeit with a community license. Are there some unique selling points for this library?

File: README.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 * [FindBugs](http://findbugs.sourceforge.net/) - Static analysis of bytecode to find potential bugs.
 * [PMD](http://pmd.sourceforge.net/) - Source code analysis of bad coding practices.
 * [SonarQube](http://www.sonarqube.org/) - Integrates other analysis components via plugins and provides an overview of the metrics over time.
+* [stan4j](http://stan4j.com/) - Commercial, structure analysis tool for Java.
```
Comment (akullpp) [inline README.md]: You should add, that there is also a free community license.

Comment (IRus) [inline README.md]: Yep, sure

**Comment by IRus** [comment]
Look at these pages:
https://github.com/EclipseDSM/Eclipse-DSM-viewer/wiki/Already-made-Java-Structure-Analysis-%28JSA%29-solutions
http://tirthalpatel.blogspot.com/2014/01/static-code-analyzers-preface.html

**Comment by akullpp** [comment]
The links don't seem to highlight anything special about stan4j. They even suggest to use one of the other tools. If there is more need for discussion or more people want it to be added, we can reopen it.

**Pull request closed without merging**